helm-charts/splunk-otel-collector/templates/operator/_helpers.tpl

```diff
@@ -181,3 +181,12 @@ Helper for generating environment variables for each instrumentation library.
     {{- printf "\n" -}}
   {{- end }}
 {{- end }}
+
+{{/*
+Helper used to define a namspace.
+- Returns namespace from a release
+- If namespaceOverride value is filled in it will replace the namespace
+*/}}
+{{- define "splunk-otel-collector.namespace" -}}
+  {{- default .Release.Namespace .Values.namespaceOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
I'd move this helper up to the helm-charts/splunk-otel-collector/templates/_helpers.tpl file.